src/WebSqlProvider.ts

```diff
@@ -80,7 +82,9 @@ export class WebSqlProvider extends SqlProviderBase.SqlProviderBase {
                 });
             }, (err) => {
                 deferred.reject(err.message + (errorDetail ? ', Detail: ' + errorDetail : ''));
-            });
+            }, () => {
+                changeVersionDeferred!!!.resolve(void 0);
```

Comment:
Added this so we can wait properly for the version change to finish. Saw an issue in UT in chrome, where the version was not actually changed at the time when we were resolving the deferred.